lib/tests/api.c

```diff
@@ -790,19 +790,19 @@ struct t_alloc_data{
 } g_alloc_data;
 
 static void* _test_alloc(void *data, size_t cb) {
-  ck_assert_ptr_eq(data, &g_alloc_data);
+  ck_assert(data == &g_alloc_data);
```

Comment:
I'd prefer this be defined at the top like the others:

```
#ifndef ck_assert_ptr_eq
#define ck_assert_ptr_eq(a,b) ck_assert((void *)(a) == (void *)(b))
#endif
```

For libcheck 0.11+, the more specific asserts give more useful error messages.

Reply:
Thank you for the comment. Yes, that was the other option. I will replace it. I did not notice these defines already on the top.